prow/cmd/deck/main.go

```diff
@@ -462,6 +464,31 @@ func handleBadge(ja *jobs.JobAgent) http.HandlerFunc {
 	}
 }
 
+func handleJobHistory(templateRoot string, ca *config.Agent, gcsClient *storage.Client) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		tmpl, err := getJobHistory(r.URL, ca.Config(), gcsClient)
```

Comment:
Set the cache headers:
```suggestion
 		setHeadersNoCaching(w)
		tmpl, err := getJobHistory(r.URL, ca.Config(), gcsClient)
```